text/metrics/0146-metrics-prototype-scenarios.md

```diff
@@ -0,0 +1,220 @@
+Resource`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md),
+so it should be covered by the metrics SDK rather than API, and strictly
+speaking it is not considered as a "dimension" from the SDK perspective.
+
+**Server temperature:**
+
+| Host Name | Temperature (F) |
+| --------- | --------------- |
+| MachineA  | 65.3            |
+
+**Server humidity:**
+
+| Host Name | Humidity (%) |
+| --------- | ------------ |
+| MachineA  | 21           |
+
+#### HTTP Server Library
+
+**Received HTTP requests:**
+
+Note: the **Client Type** is passed in via the [`OpenTelemetry
+Baggage`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/baggage/api.md),
+strictly speaking it is not part of the metrics API, but it is considered as a
+"dimension" from the metrics SDK perspective.
+
+| Host Name | Process ID | Client Type | HTTP Method | HTTP Host | HTTP Flavor | Peer IP   | Peer Port | Host IP   | Host Port |
+| --------- | ---------- | ----------- | ----------- | --------- | ----------- | --------- | --------- | --------- | --------- |
+| MachineA  | 1234       | Android     | GET         | otel.org  | 1.1         | 127.0.0.1 | 51327     | 127.0.0.1 | 80        |
+| MachineA  | 1234       | Android     | POST        | otel.org  | 1.1         | 127.0.0.1 | 51328     | 127.0.0.1 | 80        |
+| MachineA  | 1234       | iOS         | PUT         | otel.org  | 1.1         | 127.0.0.1 | 51329     | 127.0.0.1 | 80        |
+
+**HTTP server request duration:**
+
+Note: the server duration is only available for **finished HTTP requests**.
+
+| Host Name | Process ID | Client Type | HTTP Method | HTTP Host | HTTP Status Code | HTTP Flavor | Peer IP   | Peer Port | Host IP   | Host Port | Duration (ms) |
+| --------- | ---------- | ----------- | ----------- | --------- | ---------------- | ----------- | --------- | --------- | --------- | --------- | ------------- |
+| MachineA  | 1234       | Android     | GET         | otel.org  | 200              | 1.1         | 127.0.0.1 | 51327     | 127.0.0.1 | 80        | 8.5           |
+| MachineA  | 1234       | Android     | POST        | otel.org  | 304              | 1.1         | 127.0.0.1 | 51328     | 127.0.0.1 | 80        | 100.0         |
+
+### Application Requirements
+
+The application owner (developer Y) would only want the following metrics:
+
+* Server temperature - reported every 5 seconds
+* Server humidity - reported every minute
+* HTTP server request duration, reported every 5 seconds, with a subset of the
+  dimensions:
+  * Host Name
+  * HTTP Method
+  * HTTP Host
+  * HTTP Status Code
+  * Client Type
+  * 90%, 95%, 99% and 99.9% server duration
+* HTTP request counters, reported every 5 seconds:
+  * Total number of received HTTP requests
+  * Total number of finished HTTP requests
+  * Number of currently-in-flight HTTP requests (concurrent HTTP requests)
```

Comment:
I like how this example asks for three counters, because it seems possible to achieve with two instruments: a count of received requests and a histogram of response durations (i.e., seems to call for either a view or a 3rd instrument).

Reply:
And it might affect the semantic convention https://github.com/open-telemetry/opentelemetry-specification/pull/1378#discussion_r578093122.

Reply:
(Reply: https://github.com/open-telemetry/opentelemetry-specification/pull/1378#issuecomment-784910215)